charts/enterprise/authelia/values.yaml

```
@@ -267,20 +267,20 @@ authentication_backend:
     ##
     ## If your groups use the `groupOfUniqueNames` structure use this instead:
     ##    (&(uniquemember={dn})(objectclass=groupOfUniqueNames))
-    groups_filter: ""
+    groups_filter: "(member={dn})"
```

```suggestion
    groups_filter: ""
```
Also no default needed in values.yaml see notes